### PR TITLE
Fixing artifact s3 push

### PR DESCRIPTION
**Pull request by chayim**
Changes to circleci to push to s3 everything. Also small cleanup while here.

### CODE REVIEW

**Comment by chayim** [comment]
Examples, pushed everything to S3: https://app.circleci.com/pipelines/github/RedisGears/RedisGears/3212/workflows/2895fb81-8490-4cff-9956-73ea76f2943a/jobs/8213

File: .circleci/config.yml [hunk 1]
```diff
@@ -393,7 +368,7 @@ workflows:
             parameters:
               redis_version: ["6.0.9", "6.2.6"]
       - build-platforms:
-          <<: *on-integ-and-version-tags
+          <<: *on-any-branch
```
Comment (MeirShpilraien) [inline .circleci/config.yml:371]: Shouldn't we revert it?

**Review by MeirShpilraien — COMMENTED** [review]
Small question

File: .circleci/config.yml (hunk 1 above)
Comment (chayim) [reply .circleci/config.yml]: Nice catch resolved.

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**